lib/base-compiler.js

```diff
@@ -462,7 +462,7 @@ Compile.prototype.postProcess = function (result, outputFilename, filters) {
                     const postCommand = 'cat "' + outputFilename + '" | ' + postProcess.join(" | ");
                     return this.exec("bash", ["-c", postCommand], {maxOutput: maxSize})
                         .then((postResult) => {
-                            return this.handlePostProcessResult(result, postResult);
+                            return this.handlePostProcessResult(result, postResult, outputFilename);
```

Comment:
Have you checked if this changes something else unintendedly? I'll also look into it but would be great to hear if you've checked it (No worries if ou haven't!)